### ISSUE TITLE
Typo on Duplicacy.com Buy Page

### ISSUE THREAD

**Issue by TheLister** [gilbertchen/duplicacy#402]
Thanks for supporting such a great tool! I am in the process of buying the paid GUI version of Duplicacy, but I found a typo on your buy page that should be an easy fix. 

`You can alway renew a license (at the rate for subsequent years) before or after it expires.`
should probably be
`You can always renew a license (at the rate for subsequent years) before or after it expires.`

https://duplicacy.com/buy.html

**Comment by gilbertchen** [comment]
Thanks for the catch!  I should have run a spell check after ever change.

**Comment by JonasLTU** [comment]
@gilbertchen , could you please clarify, if CLI version for private use is free? In website it says that even personal is licenced and in github manual it says that it is free.

**Comment by gilbertchen** [comment]
@JonasLTU yes, the CLI version is free for personal use.  On https://duplicacy.com/buy.html, the CLI license actually means the CLI commercial license since it is stated there that `The CLI version is free for personal use so there are no personal licenses for the CLI version`.

**Comment by jonaslt** [comment]
Great. Then I will give it a roll to test how it performs under real life situations!
